src/main/java/net/dv8tion/jda/client/entities/impl/GroupImpl.java

```diff
@@ -44,6 +45,9 @@
     private String name;
     private String iconId;
     private long lastMessageId;
+	private GroupManager manager;
+	
+	private final Object mngLock = new Object();
```

Comment:
Is there a reason every entity has two xManagerUpdatable objects? (There's the one in xManager and then another one in the xImpl class)

Why doesn't getManagerUpdatable just return the one that already exists within xManager? I'm not criticizing, I just want to know if there's something I'm not understanding